lib/bolt/transport/winrm/connection.rb

```diff
@@ -31,20 +31,30 @@ def initialize(target, transport_logger)
         HTTPS_PORT = 5986
 
         def connect
-          if target.options['ssl']
+          if target.options['realm']
+            scheme = 'http'
+            transport = :kerberos
```

Comment:
Will have to verify for sure, but Kerberos should actually work fine over HTTP or HTTPS, and we'll want to allow for both.